core/deco.c

```diff
@@ -153,8 +153,8 @@ const double vpmb_conservatism_lvls[] = { 1.0, 1.05, 1.12, 1.22, 1.35 };
 
 #define TISSUE_ARRAY_SZ sizeof(ds->tissue_n2_sat)
 
-int sumx, sum1;
-long sumxx;
+int  sum1;
+long sumx, sumxx;
```

Comment:
Without having looked at the code: On many platforms `sizeof(long)==sizeof(int)`. If you need 64-bit precision, perhaps better use something like `int64_t`?